import/popups/WebrtcPermissionDialog.qml

```diff
@@ -0,0 +1,159 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+UserPrompt {
+    id: dialog
+
+    property string host
+    property var devices
+
+    property alias rememberValue: remember.checked
+    property var choices: {}
+
+    //: Allow the server to use media devices
+    //% "Allow"
+    acceptText: qsTrId("sailfish_components_webview_popups-he-allow_media_devices")
+
+    //: Deny the server access to media devices
+    //% "Deny"
+    cancelText: qsTrId("sailfish_components_webview_popups-he-deny_media_devices")
+
+    readonly property var dictionary: {
+        //% "Front camera"
+        "secondary": qsTrId("sailfish_components_webview_popups-la-primary_camera"),
+        //% "Rear camera"
+        "primary": qsTrId("sailfish_components_webview_popups-la-secondary_camera"),
+        //% "Integrated microphone"
+        "Integrated microphone": qsTrId("sailfish_components_webview_popups-la-integrated_microphone")
+    }
+
+    function translate(word) {
+        var translated = dictionary[word]
+        return translated ? translated : word
```

Comment:
I may be wrong, but I don't think `word` should be capitalized. It is an ID of media hardware from gecko. In general case it can contain, for example the model name of UVC camera or something and will be less readable if capitalized, I believe. It would be nice to be able to make the dictionary hardware dependent, but I don't know how.

Reply:
Okay, fair enough, I was expecting something like `microphone` or `camera`. In this case you are right, better to leave them as they are.